google/api_core/_python_package_support.py

```diff
@@ -132,10 +153,15 @@ def warn_deprecation_for_versions_less_than(
         or not minimum_fully_supported_version
     ):  # pragma: NO COVER
         return
+
     dependency_version = get_dependency_version(dependency_import_package)
     if not dependency_version.version:
         return
-    if dependency_version.version < parse_version(minimum_fully_supported_version):
+    # Parse the minimum required version using the new custom function
```

Comment:
```suggestion
```

"New" custom function should not be in code comments. It won't be new after today.